data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -597,7 +597,7 @@
       "tenantId": "statea"
     },
     {
```

Comment:
org admin also require indvidual search
@sanjeet1137

Reply:
@jagankumar-egov 
org_Admin was repetative .hence i udpated it with Mukta_Admin